nntrainer/compiler/tflite_interpreter.cpp

```diff
@@ -56,12 +61,135 @@ void builder2file(const flatbuffers::FlatBufferBuilder &builder,
  *
  */
 class TfOpNode {
```

Comment:
do you plan to use this node loading the tflite file as well?

Reply:
I was but I might not as we decided to use tflite directly. Things are tentative regarding loading yet.

Reply:
ok.

Reply:
Alright. I'll try to migrate to tf data structure if possible.